data/net/minecraft/client/gui/screens/recipebook/GhostRecipe.mapping

```diff
@@ -8,6 +8,8 @@ CLASS net/minecraft/client/gui/screens/recipebook/GhostRecipe
 	METHOD render (Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/Minecraft;IIZF)V
 		ARG 1 poseStack
 		ARG 2 minecraft
+		ARG 3 leftPos
+		ARG 4 topPos
```

Comment:
What would be better here: these names, or `x` and `y`?

Reply:
I'm not too sure, on the one hand it seems like `x` and `y` would be simpler, however every time this method is called the name of the fields that are used are `leftPos` and `topPos`.

For example, in `net/minecraft/client/gui/screens/inventory/CraftingScreen.java` the `render` method uses `this.leftPos` and `this.topPos`. _(This naming also occurs in `AbstractFurnaceScreen.java` and `InventoryScreen.java`)_
![image](https://user-images.githubusercontent.com/32211101/162414963-1472e258-7daa-4601-a668-d1b3334931e3.png)

_The method `renderGhostRecipe` just calls the method `render` from `net/minecraft/client/gui/screens/recipebook/GhostRecipe.java`._
![image](https://user-images.githubusercontent.com/32211101/162415320-4a39a660-368f-4a94-af49-a54857d6d924.png)